sdk/data/azcosmos/cosmos_container.go

```diff
@@ -471,6 +511,9 @@ func (c *ContainerClient) PatchItem(
 	itemId string,
 	ops PatchOperations,
 	o *ItemOptions) (ItemResponse, error) {
+	var err error
+	ctx, endSpan := runtime.StartSpan(ctx, "ContainerClient.PatchItem", c.database.client.internal.Tracer(), nil)
```

Comment:
We should also populate attributes according to semantic convention on this and other spans, see  https://github.com/open-telemetry/semantic-conventions/blob/v1.27.0/docs/database/cosmosdb.md. 

I think it could also be done as a follow up. /cc @jcocchi

Reply:
Properties are now wired